Formula/kafka.rb

```diff
@@ -1,5 +1,5 @@
 class Kafka < Formula
-  desc "Publish-subscribe messaging rethought as a distributed commit log"
+  desc "Apache Kafka is an open-source distributed event streaming platform"
```

Comment:
Usually, we don't use a formula name in a description.

```suggestion
  desc "Open-source distributed event streaming platform"
```